src/node/NodeWebSocketTransport.ts

```diff
@@ -16,17 +16,16 @@
 import { ConnectionTransport } from '../common/ConnectionTransport.js';
 import NodeWebSocket from 'ws';
 
+export const version = '13.2.0-post';
 export class NodeWebSocketTransport implements ConnectionTransport {
   static create(url: string): Promise<NodeWebSocketTransport> {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const pkg = require('../../../../package.json');
     return new Promise((resolve, reject) => {
       const ws = new NodeWebSocket(url, [], {
```

Comment:
can we `await import('../../../../package.json')` here and make the callback async? I think we might want allow json imports in TypeScript if it's not allowed? I'd prefer not to duplicate the version and have manually updated. Currently it's automatically managed by the release process.

Reply:
We can't because `json` imports are experimental in Node. Let's discuss this internally.

Reply:
Could we perhaps use `JSON.parse(fs.readFileSync())`?